src/lib/y2storage/storage_manager.rb

```diff
@@ -148,23 +148,37 @@ def deactivate
       @storage.deactivate
     end
 
-    # Probes all storage devices.
-    #
-    # Invalidates the probed and staging devicegraph. Real probing is
-    # only performed when the instance is not for testing.
-    #
-    # With the default probe callbacks, the errors reported by libstorage-ng are stored in the
-    # {#probe_issues} list.
+    # Probes all storage devices
     #
     # If this method returns false, #staging and #probed could be in bad state
     # (or not be there at all) so they should not be trusted in the subsequent
     # code.
     #
+    # @see #probe!
+    #
     # @param probe_callbacks [Callbacks::Probe, nil]
     # @return [Boolean] whether probing was successful, false if libstorage-ng
     #   found a problem and the corresponding callback returned false (i.e. it
     #   was decided to abort due to the error)
     def probe(probe_callbacks: nil)
+      probe!(probe_callbacks: probe_callbacks)
+      true
+    rescue Storage::Exception, Yast::AbortException
+      false
```

Comment:
I expect that if it is storage exception it will print to log some details, not?